### PR TITLE
fix: Do not send SDK errors for unsupported log levels

### PR DESCRIPTION
**Pull request by Mmarzex**
When I pushed out the newest SDK version to dev I noticed quite a few more SDK Caught Errors coming in and it appears to be sending an error for every info log.

### CODE REVIEW

File: node/packages/sdk/lib/structured-log-to-event.js [hunk 1]
```diff
@@ -132,6 +132,11 @@ module.exports.attemptParseStructuredLogAndCapture = (logLine) => {
       case 'WARN':
         handleWarningLog(logLineParsed);
         return;
+      case 'INFO':
```
Comment (medikoo) [inline node/packages/sdk/lib/structured-log-to-event.js]: I think there's no way we can have `INFO` returned here (?)

**Review by medikoo — COMMENTED** [review]
Thanks @Mmarzex actually it's a fix to bug I've introduced. It was supposed to have `case null` I  forgot to add it.

File: node/packages/sdk/lib/structured-log-to-event.js (hunk 1 above)
Comment (Mmarzex) [reply node/packages/sdk/lib/structured-log-to-event.js]: You're right. I thought we were returning `info` or null.

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**